src/decoration/decoration.ts

```diff
@@ -0,0 +1,33 @@
+import * as vscode from 'vscode';
+import * as path from 'path';
+import * as fs from 'fs';
+import { getNotes } from '../note-db';
+
+const decorationType = vscode.window.createTextEditorDecorationType({
+    backgroundColor: new vscode.ThemeColor('codeAnnotaion.mainColor')
+});
+
+export const setDecorations = (): void => {
+    const openEditor = vscode.window.visibleTextEditors[0];
```

Comment:
If the user has some files side by side, this will cause the decoration to only show in one of those.
The ideal would be to go over all visible editors, applying the decorations.